js/main.js

```diff
@@ -1 +1,94 @@
+const MAX_PHOTOS = 25;
+const MIN_LIKES = 15;
+const MAX_LIKES = 200;
+const MIN_COMMENTS = 0;
+const MAX_COMMENTS = 30;
+const MIN_AVATARS = 1;
+const MAX_AVATARS = 6;
 
+//getting random number from stated range
+const getRandomNumber = (a, b) => {
+	const lower = Math.ceil(Math.min(a, b));
+	const upper = Math.floor(Math.max(a, b));
+	const result = Math.random() * (upper - lower + 1) + lower;
+	return Math.floor(result);
+};
+
+//getting random element from stated array
+const getRandomArrayEllement = (elements) => elements[getRandomNumber(0, elements.length - 1)];
+
+//getting random avatars for commentators
+const getRandomAvatars = () => {
+	const avatars = [];
+	while (avatars.length <= MAX_AVATARS - 1) {
+		avatars.push(`img/avatar-${getRandomNumber(MIN_AVATARS, MAX_AVATARS)}.svg`);
+	}
+	return getRandomArrayEllement(avatars);
+};
+
```

Comment:
Создавать одинаковый массив на каждый вызов фукнции дорого
```suggestion
const avatars = Array.from({length: MAX_AVATARS}, (_, i) => `img/avatar-${i + MIN_AVATARS}.svg`);
/** getting random avatars for commentators */
const getRandomAvatars = () => getRandomArrayElement(avatars);
```